test/branchchecks_testing.jl

```diff
@@ -36,18 +36,16 @@ end
              (from=0.00337, to=00.00337), 240.0, (min=-1.17, max=1.17))
     ]
 
-    PowerSystems.check_angle_limits!(branches_test)
+    [PowerSystems.check_angle_limits!(b) for b in branches_test]
```

Comment:
I know it doesn't matter here because branches_test is small, but I wonder if you prefer the pattern of foreach instead of this, which doesn't allocate a new array.  The syntax is not exactly trivial, though.

`foreach(x -> PowerSystems.check_angle_limits!(x), branches_test)`